aws/redshift/tables/school_activity_stats.sql

```diff
@@ -37,18 +41,24 @@ CREATE table analysis.school_activity_stats AS
          ss.students,
          sc.latitude,
          sc.longitude,
+         -- teacher counts
          COUNT(DISTINCT u.id) teachers,
          COUNT(DISTINCT CASE WHEN u_students.current_sign_in_at >= dateadd (day,-364,getdate ()::DATE) THEN se.user_id ELSE NULL END) teachers_l365,
          COUNT(DISTINCT CASE WHEN u_students.current_sign_in_at >= dateadd (day,-364,getdate ()::DATE) AND se.script_id IN (1,17,18,19,23,236,237,238,239,240,241,258,259) THEN se.user_id ELSE NULL END) teachers_csf_l365,
+         COUNT(DISTINCT CASE WHEN u_students.current_sign_in_at >= dateadd (day,-364,getdate ()::DATE) AND se.script_id IN (223, 187, 181, 169, 221, 189) THEN se.user_id ELSE NULL END) teachers_csd_l365,          
          COUNT(DISTINCT CASE WHEN u_students.current_sign_in_at >= dateadd (day,-364,getdate ()::DATE) AND se.script_id IN (122,123,124,125,126,127) THEN se.user_id ELSE NULL END) teachers_csp_l365,
-         COUNT(DISTINCT CASE WHEN rpd.user_id IS NOT NULL AND rpd.course = 'CS Principles' THEN u.id ELSE NULL END) teachers_csp_pd,
-         COUNT(DISTINCT CASE WHEN rpd.user_id IS NOT NULL AND rpd.course = 'CS Discoveries' THEN u.id ELSE NULL END) teachers_csd_pd,
-         COUNT(DISTINCT f.student_user_id) students,
+         -- pd'd teacher counts
+         COUNT(DISTINCT CASE WHEN ttf.user_id IS NOT NULL THEN u.id ELSE NULL END) teachers_csf_pd,        
+         COUNT(DISTINCT CASE WHEN ttpd.studio_person_id IS NOT NULL AND ttpd.course = 'CS Discoveries' THEN u.id ELSE NULL END) teachers_csd_pd,
```

Comment:
Switch to `THEN ttpd.studio_person_id` to not count a teacher twice if they have two accounts?